Examples/Scripts/MaterialMapping/material_comparison.py

```diff
@@ -1,239 +1,139 @@
-import ROOT
-import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import uproot
 import math
 
-from ROOT import (
-    TCanvas,
-    TFile,
-    TTree,
-    gDirectory,
-    gStyle,
-    TH1D,
-    TH2F,
-    TProfile,
-    TRatioPlot,
-)
-
-
-def TH1D_from_TProf(tprof):
-    h = TH1D(
-        tprof.GetName() + "_th1",
-        tprof.GetTitle(),
-        tprof.GetNbinsX(),
-        tprof.GetXaxis().GetXmin(),
-        tprof.GetXaxis().GetXmax(),
+# Make a file list, e.g.
+files = [
+    (
+        "geant4_material_tracks.root:material-tracks",
+        "Geant4",
+        "blue",
+        "blue",
+        "Blues",
+        3,
+    ),
+    (
+        "acts_itk_validation_tracks.root:material-tracks",
+        "Acts (try all)",
+        "orange",
+        "none",
+        "Oranges",
+        4,
+    ),
+    (
+        "acts_itk_propagation_tracks.root:material-tracks",
+        "Acts (nav)",
+        "green",
+        "none",
+        "Greens",
+        5,
+    ),
+]
+
+fig, axs = plt.subplots(2, 1, sharex=True)
+fig.subplots_adjust(hspace=0.05)
+
+x_range = (-4.0, 4.0)  # [-math.pi, math.pi ] #
+x_bins = 80
+x_val_name = "v_eta"  # v_phi' #eta'
+y_val_name = "t_X0"
+
+y_lim = 0
+
+y_ratio_values = []
+y_ratio_errors = [0.0 for i in range(x_bins)]
+
+for ifile, (fname, label, color, mfc, cmap, msize) in enumerate(files):
+
+    # labels += [ label+" mean" ]
+    tree = uproot.open(fname)
+
+    # fig, ax = plt.subplots()
+    y_arr = np.array(tree[y_val_name].array())
```

Comment:
```suggestion
    y_arr = tree[y_val_name].array(library="np")
```
I think that should work as well, similar for the other cases

Reply:
Ok, thanks, will change.